modyn/trainer_server/internal/trainer/remote_downsamplers/remote_gradnorm_downsampling.py

```diff
@@ -90,3 +90,10 @@ def select_points(self) -> tuple[list[int], torch.Tensor]:
 
         selected_ids = [self.index_sampleid_map[sample] for sample in downsampled_idxs]
         return selected_ids, weights
+
+    @property
+    def requires_grad(self) -> bool:
+        if isinstance(self.per_sample_loss_fct, torch.nn.CrossEntropyLoss):
+            return False
```

Comment:
In the `self.get_scores` method, now we can remove the `with torch.inference_mode()`

Reply:
Done